lib/commands/migrate.js

```diff
@@ -180,7 +180,7 @@ function runMigration(currentConfigPath, outputConfigPath) {
 					}
 					console.log(
 						chalk.green(
-							`\n ✔︎ New webpack v2 config file is at ${outputConfigPath}`
+							`\n ✔︎ New webpack config file is at ${outputConfigPath}`
 						)
```

Comment:
Since how plugins and loaders works changed from v3->v4, i think we could add a message down below, when everything is finished, telling the user to update all plugins/loaders. What do you all think?

Reply:
Definitely, we should add that.